docs/launching_enveloping.md

```diff
@@ -27,7 +27,7 @@ In order to run an instance of Enveloping in Regtest:
     ```json5
     {
         url: 'localhost', // the interface where the relay server will be exposed
-        port: 8090, // the port where it will be running
+        port: 8095, // the port where it will be running
```

Comment:
changes to this file should be undone, as these instructions are unrelated to the tests.

in fact, suggesting that the server should be launched in this port will cause the original problem to re-appear

Reply:
Fixed.

Reply:
please apply the same correction to the rest of the file